kairon/shared/channels/mail/data_objects.py

```
@@ -0,0 +1,125 @@
+
+    @staticmethod
+    def soft_delete_doc(bot: str, intent: str):
+        try:
+            MailClassificationConfig.objects(bot=bot, intent=intent).update(status=False)
+        except Exception as e:
+            raise AppException(str(e))
+
+    @staticmethod
+    def update_doc(bot: str, intent: str, **kwargs):
+        keys = ['entities', 'subjects', 'classification_prompt', 'reply_template']
+        for key in kwargs.keys():
```

Ruff (0.7.0)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)